app/src/main/java/com/example/fittracker/ConstantUtils.java

```diff
@@ -19,4 +19,14 @@
     public static final String DATAHANDLER_TYPE = "text/plain";
     public static final Character COMA = ',';
     public static final String MAIL_HOST = "smtp.gmail.com";
+    public static final String WEATHER_BASE_URL = "https://api.openweathermap.org";
+    public static final int LOCATION_REQUEST_CODE = 1;
+    public static final String MARVEL_API_KEY_PUBLIC = "08cd7f2cd10462f4d20ab91d76ecec33";
```

Comment:
why do you have the MAVEL api keys?

Reply:
They must be used in the following exercise in the PPS plan